src/transforms/group.d.ts

```diff
@@ -29,7 +29,7 @@ export interface GroupOutputOptions<T = Reducer> {
    * Plot.groupX({y: "count", sort: "count"}, {fill: "sex", x: "sport"})
    * ```
    */
-  sort?: T | null;
+  sort?: ReducerSort | null;
```

Comment:
This is incorrectly narrowing the type of `T` (_e.g._, in the case that `T` is `BinReducer` instead of `Reducer`). We’ll need to find another way to type this.